src/data.rs

```diff
@@ -195,6 +197,12 @@ fn set_matching_profiles(
 pub fn get_all_devices_of_profile(devices: &ListOfDevicesT, profile: &Profile) -> Vec<usize> {
     let mut found_indices = vec![];
 
+    let re: Option<Regex> = if let Some(dev_pattern) = &profile.device_name_pattern {
+        Some(Regex::new(dev_pattern).expect("Failed to initialize regex"))
```

Comment:
Does this mean we should to write patterns in config according to Rust?

Reply:
Yes